model_compression_toolkit/core/common/mixed_precision/kpi_tools/kpi_data.py

```diff
@@ -15,19 +15,23 @@
 from typing import Callable, Any
 import numpy as np
 
-from model_compression_toolkit import FrameworkInfo, KPI, MixedPrecisionQuantizationConfig, CoreConfig
+from model_compression_toolkit import FrameworkInfo, KPI, CoreConfig
 from model_compression_toolkit.core.common import Graph
+from model_compression_toolkit.core.common.constants import FLOAT_BITWIDTH
 from model_compression_toolkit.core.common.framework_implementation import FrameworkImplementation
+from model_compression_toolkit.core.common.graph.edge import EDGE_SINK_INDEX
 from model_compression_toolkit.core.common.target_platform import TargetPlatformCapabilities
 from model_compression_toolkit.core.runner import read_model_to_graph, get_finalized_graph
+from model_compression_toolkit.core.common.logger import Logger
 
 
 def compute_kpi_data(in_model: Any,
                      representative_data_gen: Callable,
                      core_config: CoreConfig,
                      tpc: TargetPlatformCapabilities,
                      fw_info: FrameworkInfo,
-                     fw_impl: FrameworkImplementation) -> KPI:
+                     fw_impl: FrameworkImplementation,
+                     bops_kpi: bool = True) -> KPI:
```

Comment:
We need this for Pytorch framework because BOPS KPI is currently not supported for it.
Do you think it would be sufficient to add a TODO to remove this flag once we implement BOPs KPI in Pytorch?

Reply:
Sure. Add a TODO for now, please